src/app-header.c

```diff
@@ -134,11 +134,15 @@ void app_header_init(AppHeader* ah)
 	g_signal_connect_swapped(prev, "clicked", (GCallback) &app_header_nav_prev, ah);
 	g_signal_connect_swapped(ah->btn_current, "clicked", (GCallback) &app_header_nav_current, ah);
 	g_signal_connect_swapped(ah->btn_next, "clicked", (GCallback) &app_header_nav_next, ah);
+	gtk_widget_set_tooltip_text(prev, "Previous");
+	gtk_widget_set_tooltip_text(ah->btn_current, "Current");
```

Comment:
Previous week (In the future there will probably be a month view, but let's deal with that when it happens)

Reply:
^ obviously should have been on the previous line.

"Current" isn't that clear either. Maybe "Jump to current week" or "Show today"?